packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/queued_item_store/dart_queued_item_store.vm.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'dart:async';
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/drift/drift_queued_item_store.dart';
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/queued_item_store.dart';
+import 'package:aws_common/aws_common.dart';
+import 'package:meta/meta.dart';
+
+/// {@macro amplify_analytics_pinpoint_dart.dart_queued_item_store}
+class DartQueuedItemStore implements QueuedItemStore, Closeable {
+  /// {@macro amplify_analytics_pinpoint_dart.dart_queued_item_store}
+  DartQueuedItemStore(String? storagePath)
+      : _database = DriftQueuedItemStore(storagePath!);
+
+  final DriftQueuedItemStore _database;
+
+  @override
+  Future<void> add(String string) {
+    return _database.add(string);
+  }
+
+  @override
+  Future<void> deleteItems(Iterable<QueuedItem> items) {
+    return _database.deleteItems(items);
```

Comment:
Ok well we can't use delete as it's another method so we would have to switch to Items

Reply:
Where? I don't see `delete` on `QueuedItemStore`

Reply:
To clarify I switched to: 
addItems/getItems/deleteItems

We can't use delete because in ```
DriftQueuedItemStore extends _$DriftQueuedItemStore
    implements QueuedItemStore 
```
The delete method name is used in the superclass.

Reply:
Why does Drift DB need to implement `QueuedItemStore`?

Reply:
Would prefer `add`/`getCount`/`delete` as suggested

Reply:
the current api names are:

`addItem` `getCount` and `deleteItems`

Drift DB implements `QueuedItemStore` for uniformity as its purpose is to store queued items.   

I don't think we should make it no longer implement that interface just so that we can name an API method `delete` instead of `deleteItems`.  Furthermore while `delete` methods normally remove a single item the `deleteItems` is specifically meant to delete multiple items at once.